File: PeliculasITIC91/PeliculasITIC91/Data/Pelicula.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PeliculasITIC91.Data
{
    public class Pelicula
    {
        public long Id { get; set; }
        public string Titulo { get; set; }
        public string Genero { get; set; }
        public int Duracion { get; set; }

        public int Anio { get; set; }

        public override string ToString()
        {
            return "Genero: " + Genero + "  Duracion: " + Duracion + " min.  Año: " + Anio;

        }

    }
}

File: Api/models/Pelicula.js
'use strict';

const { Sequelize } = require('sequelize');
const sequelize = new Sequelize({
    dialect: 'sqlite',
    storage: './storage/database.sqlite'
});

 
const Pelicula = sequelize.define('Pelicula', {
    id: {
        primaryKey: true,
        type: Sequelize.BIGINT,
        autoIncrement: true
    },
    titulo: {
        type: Sequelize.STRING
    },
    genero: {
        type: Sequelize.STRING
    },
    duracion: {
        type: Sequelize.BIGINT
    },
    anio: {
        type: Sequelize.BIGINT
    }

});

Pelicula.sync();

module.exports = Pelicula;